pkg/apis/ext.cattle.io/v1/types.go

```diff
@@ -35,7 +35,12 @@ type UserActivityStatus struct {
 
 // Token is used to authenticate requests to Rancher.
 type Token struct {
-	metav1.TypeMeta   `json:",inline"`
+	// Standard kubernetes type meta information
+	metav1.TypeMeta `json:",inline"`
+
+	// Standard kubernetes object meta information.
+	// BEWARE of non-standard behaviour: Name and GenerateName are not respected.
+	// A name is generated with a predefined prefix instead ('token-').
```

Comment:
I'd keep it aligned with core types and types that are already part of the public api (e.g. Project)
```suggestion
	metav1.TypeMeta `json:",inline"`

	// Standard object's metadata.
	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
	// Note: Name and GenerateName are not respected. A name is always generated with a predefined prefix.
	// +optional
```
See https://github.com/kubernetes/kubernetes/blob/fc3e0cf60d6f0e3c58000e0b4181dad76f99e9ed/staging/src/k8s.io/api/core/v1/types.go#L5122-L5126
Note that metav1.TypeMeta doesn't have a doc comment.

Reply:
Will do.

Reply:
done. commit 75f7ea8310bb303148c2b5a195877dcc76e172ad